consumer/src/test/kotlin/integrations/KafkaEventSinkIT.kt

```diff
@@ -0,0 +1,103 @@
+package integrations
+
+import kotlinx.coroutines.experimental.GlobalScope
+import kotlinx.coroutines.experimental.delay
+import kotlinx.coroutines.experimental.launch
+import kotlinx.coroutines.experimental.runBlocking
+import org.apache.kafka.clients.admin.AdminClient
+import org.apache.kafka.clients.admin.NewTopic
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.ByteArraySerializer
+import org.apache.kafka.common.serialization.LongSerializer
+import org.codehaus.jackson.map.ObjectMapper
+import org.junit.After
+import org.junit.Before
+import org.junit.ClassRule
+import org.junit.Test
+import org.neo4j.graphdb.GraphDatabaseService
+import org.neo4j.graphdb.Label
+import org.neo4j.graphdb.Node
+import org.neo4j.test.TestGraphDatabaseFactory
+import org.testcontainers.containers.KafkaContainer
+import streams.StreamsEventSinkQueryExecutionTest
+import java.util.*
+import java.util.concurrent.ThreadLocalRandom
+import kotlin.test.assertEquals
+
+class KafkaEventSinkIT {
+    companion object {
+        /**
+         * Kafka TestContainers uses Confluent OSS images.
+         * We need to keep in mind which is the right Confluent Platform version for the Kafka version this project uses
+         *
+         * Confluent Platform | Apache Kafka
+         *                    |
+         * 4.0.x	          | 1.0.x
```

Comment:
should we have a parameterized test for all 3 platforms?

Reply:
created #51